config/crd/kustomization.yaml

```diff
@@ -6,26 +6,28 @@ resources:
 - bases/addons.cluster.x-k8s.io_helmreleaseproxies.yaml
 #+kubebuilder:scaffold:crdkustomizeresource
 
-patchesStrategicMerge:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
```

Comment:
I'm going to take another pass at the `kustomization.yaml` files and see if some comments should be removed (or preserved), because this regeneration made them less organized (although still valid).